### PR TITLE
ts2gas in TypeScript

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
[x] `npm run build`
[x] `npm run test`
[x] `npm run lint`

### CODE REVIEW

**Comment by grant** [comment]
Can you add Travis?
- https://github.com/google/clasp/blob/master/.travis.yml

**Comment by PopGoesTheWza** [comment]
I do not know travis, yaml and all that CI nonsense ;)

I'll try though

**Comment by PopGoesTheWza** [comment]
@grant likely my travis is flawed

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- '10'
+- '8'
+- '6'
+cache:
+  directories:
+  - "$HOME/.npm"
+before_install:
+- npm install -g npm@latest
+install:
+- npm ci
```
Comment (grant) [inline .travis.yml]: I think we should do:
- yarn

**Comment by PopGoesTheWza** [comment]
This is somewhat beyond my current knowledge. I'll give it a look later

**Comment by grant** [comment]
I'll merge and fix the Travis. Thanks for the PR!

**Pull request merged**